src/main/java/duke/other/DateValidator.java

```diff
@@ -0,0 +1,49 @@
+package duke.other;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a date validator to check is a date is formatted in a specified format. A DateValidator object
+ * corresponds to a Date Formatter of a specific date format.
+ */
+public class DateValidator {
+    private DateTimeFormatter dateFormatter;
+
```

Comment:
why the extra line haha you can use IntelliJ to format everything nicely for you before you commit code!